apps/mark-scan/frontend/src/pages/replace_jammed_sheet_screen.tsx

```diff
@@ -55,6 +56,7 @@ export function ReplaceJammedSheetScreen(
       icon={<Icons.Info />}
       title="Jam Cleared"
       voterFacing={false}
+      buttons={<ResetVoterSessionButton />}
```

Comment:
Is there test coverage for this flow that should get updated?

Reply:
No specific flow tests, currently (that I know of)

Reply:
Cool, if you have time/energy to add one, that could be nice, but I know it's kind of annoying to add tests for uncovered code so no pressure

Reply:
Ah not really; mostly that there didn't seem to be a useful test to add (given the type-checked constraints on the page component and the tests for the state machine and reset button) - could probably add a render check to verify the button's being rendered, though